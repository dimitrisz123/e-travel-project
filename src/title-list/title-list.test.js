import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import TitleList from "./title-list";

it("renders TitleList correctly", () => {
	const filteredTitle = [
		{
			fields: {
				episode_id: "1",
				title: "a new hope",
				release_date: "18-7-1999"
			}
		}
	];
	const wrapper = shallow(<TitleList filteredTitle={filteredTitle} />);
	expect(toJson(wrapper)).toMatchSnapshot();
});
