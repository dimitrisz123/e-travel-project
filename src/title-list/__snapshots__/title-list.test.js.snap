// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TitleList correctly 1`] = `
<div
  className="title-list"
>
  <Title
    episodeNumber="1"
    key="1"
    releaseDate="18-7-1999"
    title="a new hope"
  />
</div>
`;
